src/test/java/com/google/sps/AttendeeTest.java

```diff
@@ -0,0 +1,50 @@
+import java.util.Date;
+import java.text.DateFormat;
+import java.text.SimpleDateFormat;
+import java.text.ParseException;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.tools.development.testing.LocalServiceTestHelper;
+import com.google.appengine.tools.development.testing.LocalDatastoreServiceTestConfig;
+import org.junit.After;
+import org.junit.Before;
+import com.google.sps.data.Attendee;
+
+@RunWith(JUnit4.class)
+
+public class AttendeeTest {
+
+  private final LocalServiceTestHelper helper =
+  new LocalServiceTestHelper(new LocalDatastoreServiceTestConfig());
+
+  @Before
+  public void setUp() {
+   helper.setUp();
+  }
+
+  @After
+  public void tearDown() {
+   helper.tearDown();
+  }
+
+  @Test
+  public void attendeeTester() {
+    
+    // Creates test data. 
+    Date date = new Date();
```

Comment:
This doesnt need to be a separate creation. You dont use it anywhere else.
new Attendee(..., new Date()) should work just fine.

Reply:
It is used to compare to the getter in line 45

Reply:
ah, right, okie.